e2e/content-services/components/permissions-component.e2e.ts

```diff
@@ -1,413 +0,0 @@
-/*!
- * @license
- * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { PermissionsPage } from '../../content-services/pages/permissions.page';
```

Comment:
After these changes is PermissionsPage used somewhere or can we remove. whole file? If it is used then maybe we can clean up that file if something is unused there?

Reply:
yeah, removing the whole page and also the permission dialog page